perceval/backends/mozilla/remo.py

```diff
@@ -22,6 +22,7 @@
 
 import json
 import logging
+from urllib.parse import urlparse, parse_qs
```

Comment:
I'd rather import `urllib.parse` and use these functios as `urllib.parse.urlparse(next_uri)` and so on.
This is the style we're following in perceval when we use third-party libraries.